src/main/java/com/jvm_bloggers/frontend/admin_area/blogs/BlogActionPanel.java

```diff
@@ -52,6 +55,22 @@ private AjaxButton createDeactivateBlogButton() {
         return deactivate;
     }
 
+    private AjaxButton createRequireModerationBlogButton() {
+        AjaxButton requireModeration = createBlogActionButton("requireModerationBlog",
```

Comment:
It would look much clearer if you have applied paired brackets notation described here: http://www.yegor256.com/2014/10/23/paired-brackets-notation.html.